packages/ui/src/views/chatmessage/ChatMessage.jsx

```diff
@@ -1091,7 +1262,7 @@ export const ChatMessage = ({ open, chatflowid, isDialog, previews, setPreviews
                                             <IconButton
                                                 onClick={() => onMicrophonePressed()}
                                                 type='button'
-                                                disabled={loading || !chatflowid}
+                                                disabled={loading || !chatflowid || !isLeadSaved}
```

Comment:
when you do this, isnt chat always be disabled?
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/1f171dd0-2c6d-46cf-8853-e96e97749177)

Reply:
this if microphone button, what about send button? image upload button?

Reply:
all the same issues with embedded - https://github.com/FlowiseAI/FlowiseChatEmbed/pull/119/files#r1573492973